api/src/neo4j/queries/search/globalSearch.js

```diff
@@ -286,43 +270,12 @@ const globalSearch = async ({ searchTerm, version, limit }) => {
   }, {});
 };
 
-const modelSearch = async ({ searchTerm, model, version, limit }) => {
-  const match = MODELS.filter(m => m.label == model);
-  if (match.length === 0) {
-    throw new Error(`Invalid model: ${model}`);
-  }
-
-  const results = await _search({
-    searchTerm,
-    model,
-    version,
-    limit: limit || 50,
-  });
-
-  return {
-    [match[0].name]: {
-      ...results,
-      name: match[0].name,
-      metabolite: results.metabolite.map(m => ({
-        ...m,
-        compartment: m.compartment.name,
-      })),
-    },
-  };
-};
-
 /*
  * The search consists of two steps
  * 1. Do a fuzzy search over all nodes covered by full-text search index
  * 2. Fetch results for each component type (parallelly) and return result
  */
```

Comment:
Hi @MalinAhlberg, thanks for noting this. Not sure if you have marked the right lines here but I think the following comments could be removed.

```
  // Metabolites are not included as it would mess with the limit and
  // relevant metabolites should be matched through CompartmentalizedMetabolites
```